src/effects/audiounits/AudioUnitValidator.cpp

```diff
@@ -205,10 +208,10 @@ void AudioUnitValidator::EventListener(const AudioUnitEvent *inEvent,
          // Reassign settings, so that there is "stickiness" when dialog is
```

Comment:
I think that you could remove the call to ModifySettings and leave the lambda body there, even at this stage? this because the passed EffectSettings arg is not used anymore in the lambda.